actions/universal-fix-dropbox-links.js

```diff
@@ -6,11 +6,15 @@ var xmlAssignments = [];
 var canvasAssignments = [];
```

Comment:
This shouldn't have any changes except for the `description-quicklinks` grandchild. Can we make sure these changes don't affect anything else?

Reply:
Should be fine